core/storage/base_model/gae_models.py

```diff
@@ -95,10 +95,12 @@ class BaseModel(datastore_services.Model):
     # the model's id is not used as the key for the Takeout dict.
     ID_IS_USED_AS_TAKEOUT_KEY = False
 
-    # When this entity was first created. This value should not be modified.
+    # When this entity was last updated. This value should only be modified in
+    # this class only and nowhere else.
```

Comment:
```suggestion
    # When this entity was last updated. This value should only be modified in
    # this class and nowhere else.
```

'only' is redundant.